torch_xla/distributed/fsdp/xla_fully_sharded_data_parallel.py

```diff
@@ -295,6 +295,7 @@ def __init__(
       sharding_world_size: Optional[int] = None,
       shard_param_on_dim_0: bool = False,
       pin_layout_in_collective_ops: bool = True,
+      coalesce_all_gather_ops: bool = False,
```

Comment:
Do you mind explaining the change in this file? I think `coalesce_all_gather_ops ` is always `False` in our test, did you run into these issues with your own test?